service/src/main/kotlin/app/cash/backfila/service/persistence/DbEventLog.kt

```diff
@@ -0,0 +1,68 @@
+package app.cash.backfila.service.persistence
+
+import java.time.Instant
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.GeneratedValue
+import javax.persistence.Table
+import misk.hibernate.DbTimestampedEntity
+import misk.hibernate.DbUnsharded
+import misk.hibernate.Id
+
+@Entity
+@Table(name = "event_logs")
+class DbEventLog() : DbUnsharded<DbEventLog>, DbTimestampedEntity {
+
+  enum class Type {
```

Comment:
I never reveal this anywhere but I thought it may be useful later. Maybe I should take it out, thoughts?

Reply:
I might distinguish between different kinds of errors?
Client errors vs backfila errors?

I like the idea of type.

Reply:
I thought it might be useful for adding a filter button to only show errors etc. Not sure if thats actually useful. Can you think of any other uses for it?